.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/alainmarcel/Surelog.git
 [submodule "uhdm-integration"]
 	path = uhdm-integration
-	url = https://github.com/alainmarcel/uhdm-integration.git
+	url = https://github.com/antmicro/uhdm-integration.git
```

Comment:
Please do not forget to remove this change after uhdm-integration merge.

Reply:
It is now removed